oelint_adv/__main__.py

```diff
@@ -19,6 +23,19 @@
 sys.path.append(os.path.abspath(os.path.join(__file__, '..')))
 
 
+def parse_configfile():
+    config = ConfigParser()
+    for conffile in [os.environ.get('OELINT_CONFIG', '/does/not/exist'),
+                     os.path.join(os.getcwd(), '.oelint.cfg'),
+                     os.path.join(os.environ.get('HOME', '/does/not/exist'), '.oelint.cfg')]:
+        try:
+            config.read(conffile)
+            return dict(config.items('oelint'))
+        except (FileNotFoundError, PermissionError, SystemError, NoSectionError, NoOptionError, ParsingError):
+            pass
```

Comment:
I would prefer a bit additional output for different exceptions. Silently passing them will make it hard to debug what went wrong when reading the config file.

I haven't tested it yet but something like this:

```suggestion
        except FileNotFoundError:
            pass
        except (PermissionError, SystemError) as e:
            print(e)
        except (NoSectionError, NoOptionError, ParsingError) as e:
            print(f'Failed parsing config file {conffile}. {e!r}')
```

Reply:
Done